mantidproject/mantid
# PowderISIS System Test for Polaris

fixes #14303 

> - [x] merge #14504 before this

New test created by the name of `ISISPowderDiffractionPolarisTest`

**To Test**
- Run the test locally
- Make sure all the file are cleared once the system test has passed
